DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/EditCentreWebsiteDetails.cshtml

```diff
@@ -0,0 +1,42 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem
+@model EditCentreWebsiteDetailsViewModel
+
+@{
+  var errorHasOccurred = !ViewData.ModelState.IsValid;
+  ViewData["Title"] = errorHasOccurred ? "Error: Edit Centre Website Details" : "Edit Centre Website Details";
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    @if (errorHasOccurred) {
+      <vc:error-summary order-of-property-names=
+                        "@(new[] { nameof(Model.CentreTelephone), nameof(Model.CentreEmail), nameof(Model.CentrePostcode), nameof(Model.OpeningHours), nameof(Model.CentreWebAddress), nameof(Model.OrganisationsCovered), nameof(Model.TrainingVenues), nameof(Model.OtherInformation) })"/>
+    }
+
+    <h1 id="page-heading" class="nhsuk-heading-xl">Edit centre website details</h1>
+
+    <form method="post" novalidate asp-action="EditCentreWebsiteDetails">
+
+      <vc:text-input asp-for="CentreTelephone" label="Centre telephone:" populate-with-current-value="true" type="text" spell-check="false" autocomplete="" hint-text="" css-class="nhsuk-u-width-one-half"/>

```

Comment:
I just noticed that there's inconsistency between including and not including ":" after each field. I think we should go for no colons as this is what they do in some NHS examples 🙏

Reply:
Also, could we add every param onto a new line here as the line is quite long